roles/ceph-restapi/tasks/docker/pre_requisite.yml

```diff
@@ -33,6 +33,13 @@
     state: started
     enabled: yes
 
+- name: install six
```

Comment:
Unfortunately, I need to ensure that a certain version of six is installed, so I can't use the with_items loop.  I did add a comment to indicate this.